src/nvim/eval.c

```diff
@@ -10609,6 +10612,7 @@ static void f_has(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     "find_in_path",
     "float",
     "folding",
+    "folding_fillchars",
```

Comment:
We don't need these feature names since users can check `has('nvim-0.3')`. Also avoids having to guess what Vim will call a feature.